docsite/docs/Docs/fal-cli/pure-python-models.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 8
+---
+
+# Making pure Python dbt models
```

Comment:
This section reads more like an intro to a blog post rather than a documentation section. I would only keep line 7 and clarify it.